x-pack/plugins/security_solution/server/usage/collector.ts

```diff
@@ -76,9 +76,14 @@ export const registerCollector: RegisterCollector = ({
     isReady: () => kibanaIndex.length > 0,
     fetch: async (callCluster: LegacyAPICaller): Promise<UsageData> => {
       const savedObjectsClient = await getInternalSavedObjectsClient(core);
+      const [detections, endpoints] = await Promise.allSettled([
```

Comment:
@rylnd, thoughts? Want to prevent any failures from causing the entire usage collection to fail